app/views/ogi_pilot_sessions/index.html.erb

```diff
@@ -1,62 +1,64 @@
-<div class="container">
-    <div class="row">
-        <div class="col-md-8 offset-md-2 mt-5 fancy-forms">
-            <div class="text-end">
-                <%= button_to new_ogi_pilot_session_path, method: :get, class: "btn" do%>
-                <h1><i class="bi bi-building-fill-add"></i></h1>
-                <% end %>
-            </div>
-            <ul class="nav nav-tabs  mt-3" id="myTab" role="tablist">
-                <li class="nav-item">
-                    <a class="nav-link" id="publish_subscribe" data-toggle="tab" href="#"  data-url="publish_subscribe" role="tab" aria-controls="publish_subscribe" aria-selected="false">Publish Subscribe</a>
-                </li>
-            </ul>
-            <% if @sessions.present? %>
-                <div class="container">
-                    <table class="table table-striped">
-                        <thead>
-                            <tr>
-                                <th>Session Id</th>
-                                <th>Channel</th>
-                                <th>Topic</th>
-                                <th>End Point</th>
-                                <th>Actions</th>
-                            </tr>
-                        </thead>
-                        <tbody>
-                            <% @sessions.each do |session| %>
+<turbo-frame id="session_form">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-8 offset-md-2 mt-5 fancy-forms">
+                <div class="text-end">
+                    <%= button_to new_ogi_pilot_session_path, method: :get, class: "btn" do%>
```

Comment:
BTW it is a good practice to put a title on the buttons and links so that it has something for screen readers and hovering will give you the tooltip. You can even enable the fancy bootstrap tooltips if you want.